framework/test/unit/components/storage/entities/round.js

```diff
@@ -972,7 +972,7 @@ describe('Round', () => {
 	});
 
 	describe('performVotesSnapshot()', () => {
-		afterEach(() => RoundEntity.clearVotesSnapshot());
+		afterEach(async () => RoundEntity.clearVotesSnapshot());
```

Comment:
`RoundEntity.clearVotesSnapshot()` returnes a Promise, and mocha support to handle promise returned from function. Why we had to explicitly declare it as `async`?